db/versions/0017-migration.sql

```diff
@@ -0,0 +1,42 @@
+begin
+
+  -- lock this table before reading from it, to prevent loss of concurrent
+  -- updates when the table is dropped.  Note that this may lead to concurrent
+  -- updates failing; the important thing is that they not succeed without
+  -- taking effect.  Failed updates will be retried.
+  lock table taskcluster_github_builds_entities;
+
+  create table github_builds
+  as
+    select
+      (value ->> 'organization')::text as organization,
+      (value ->> 'repository')::text as repository,
+      (value ->> 'sha')::text as sha,
+      (value ->> 'taskGroupId')::text as task_group_id,
```

Comment:
I think Dustin recommended moving away from azure's way of storing things as uuids but rather store things as slug ids. We'll want to modify the migration and downgrade script to make sure we have the proper slugid -> uuid and uuid -> slugid mappings.